.github/workflows/pull-request.yaml

```diff
@@ -5,6 +5,19 @@ on:
   pull_request: {}
 
 jobs:
+  validate_metrics_docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Golang 1.18
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.18'
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Generate metrics docs
+        run: make generate-doc
+      - name: Validate no changes
+        run: git difftool -y --trust-exit-code --extcmd=./hack/diff-csv.sh
```

Comment:
Why is the default `git diff` not enough?

Reply:
using extensions, in this case the ./hack/diff-csv.sh, allows us to ignore certain patterns that might have been changed on some compilation steps, such as the creation date on manifests

Reply:
Ok, is it currently useful? I have tried searching for string `createdAt` in the repo, but it is not anywhere. What manifests do you mean?

Reply:
Ah, you are right, I tested `make bundle` and checked the `ClusterServiceVersion` generated in the `out/` directory, and unlike, for example, HCO, SSP does not create a `createdAt` annotation. Thank you for the warning, removing